spec/services/markdown_processor/parser_spec.rb

```diff
@@ -448,4 +448,28 @@ def generate_and_parse_markdown(raw_markdown)
       expect(generate_and_parse_markdown(codeblock)).to include("%}")
     end
   end
+
+  context "with additional_liquid_tag_options" do
+    it "passes those options to Liquid::Template.parse" do
+      # rubocop:disable RSpec/VerifiedDoubles
+      #
+      # I don't want to delve into the implementation details of liquid to test what the parse
+      # method's return value.
+      parse_response = double("parse_response", render: "liquified!")
```

Comment:
if you had delved in (I don't blame your choice here) the instance_double would be a Liquid::Template instance. It answers `#render` so you're fine.

I think this would be the equivalent?

```ruby
parse_response = instance_double("Liquid::Template", render: "liquified!")
```